pkg/helmdeployer/deployer.go

```diff
@@ -405,7 +406,20 @@ func (h *Helm) install(bundleID string, manifest *manifest.Manifest, chart *char
 	if !dryRun {
 		logrus.Infof("Helm: Upgrading %s", bundleID)
 	}
-	return u.Run(releaseName, chart, values)
+	rel, err := u.Run(releaseName, chart, values)
+	if err != nil && err.Error() == HelmUpgradeInterruptedError {
+		logrus.Infof("Helm error: another operation (install/upgrade/rollback) is in progress for %s. Doing a rollback", bundleID)
```

Comment:
Don't you want to reuse HelmUpgradeInterruptError such as 
```suggestion
		logrus.Infof("Helm error: %s for %s. Doing a rollback", HelmUpgradeInterrruptError, bundleID)
```

Reply:
I feel like it's more readable without the constant, but 🤷 
@raulcabello if you change the commit, can you make the commit message start with a capital letter? :)

Reply:
@olblak suggestion didn't compile because it is `HelmUpgradeInterrruptedError`, no `HelmUpgradeInterrruptError`. I changed this and the commit message. Please review it again